test/zdtm/static/cgroup_yard.desc

```diff
@@ -0,0 +1 @@
+{'flavor': 'h', 'flags': 'suid', 'opts': '--manage-cgroups --cgroup-yard ../../../../../../external_yard'}
```

Comment:
we need a comment which explains these ../../.../../..

Reply:
can we just specify --cgroup-yard=/sys/fs/cgroup?

Reply:
No, the special test cgroup `zdtmtst` is not mounted there.